### PR TITLE
fix for issue #147

### PR DESCRIPTION
**Pull request by le-zell**

### CODE REVIEW

File: cas-security-spring-boot-autoconfigure/src/main/java/com/kakawait/spring/boot/security/cas/autoconfigure/CasHttpSecurityConfigurer.java [hunk 1]
```diff
@@ -176,7 +183,19 @@ public void init(HttpSecurity http) throws Exception {
             filterConfigurer.configure(filter);
 
             SingleSignOutFilter singleSignOutFilter = new SingleSignOutFilter();
-            singleSignOutFilter.setCasServerUrlPrefix(casSecurityProperties.getServer().getBaseUrl().toASCIIString());
+            try {
+                ConfigurationKey<String> key = ConfigurationKeys.CAS_SERVER_URL_PREFIX;
+                //Field field = FieldUtils.getField(SingleSignOutHandler.class, key.getName());
```
Comment (kakawait) [inline cas-security-spring-boot-autoconfigure/src/main/java/com/kakawait/spring/boot/security/cas/autoconfigure/CasHttpSecurityConfigurer.java:188]: We may remove that commented line?

**Review by kakawait — CHANGES_REQUESTED** [review]
Otherwise looks good

File: cas-security-spring-boot-autoconfigure/src/main/java/com/kakawait/spring/boot/security/cas/autoconfigure/CasHttpSecurityConfigurer.java (hunk 1 above)
Comment (le-zell) [reply cas-security-spring-boot-autoconfigure/src/main/java/com/kakawait/spring/boot/security/cas/autoconfigure/CasHttpSecurityConfigurer.java:188]: of course. No pb

**Review by kakawait — APPROVED** [review]

**Comment by kakawait** [comment]
@le-zell I'll release new version tonight

**Comment by le-zell** [comment]
Cool thanks


Le jeu. 26 nov. 2020 à 16:07, Thibaud Lepretre <notifications@github.com> a
écrit :

**Comment by kakawait** [comment]
@le-zell I've pushed new release. But can still took few mins even hours to be propagated inside maven central.

**Pull request merged**